### PR TITLE
ab: error when deleting course with assignments

### PR DESCRIPTION
**Pull request by annagav**

### CODE REVIEW

File: instructor/common.py [hunk 1]
```diff
@@ -211,7 +212,10 @@ def course_modify(request):
             user = User.objects.get(username=request.user)
             for obj in formset.deleted_objects:
                 if obj.owner == user and obj != assignment.course:
```
Comment (noisecapella) [inline instructor/common.py]: If the course is selected I don't see the error message when attempting to delete it. Can you add an error message for this case?

**Comment by noisecapella** [comment]
Also rebase please :)

**Comment by annagav** [comment]
Rebased!

**Comment by pdpinch** [comment]
@noisecapella when you have a few minutes, can you re-review this PR?

**Comment by noisecapella** [comment]
:+1:

**Pull request merged**